packages/cli/src/util/deploy/get-preferred-preview-url.ts

```diff
@@ -1,48 +0,0 @@
-import isWildcardAlias from '../alias/is-wildcard-alias';
-import { getCertsForCn } from '../certs/get-certs-for-cn';
-import Client from '../client';
-
-/**
- * Tries to find the "best" alias url.
- * @param aliasList
- */
-export async function getPreferredPreviewURL(
```

Comment:
This function was showing the best alias we could find, but for instant preview URLs, we want the unique deployment URL.